controllers/ingestion/http/http.go

```diff
@@ -0,0 +1,79 @@
+package http
```

Comment:
Should this package sit here? I think it can be used in future logic that might not related to Ingestion. This should be a generic Druid client

Reply:
noted !
Thanks man